api/swagger-spec/v1.json

```diff
@@ -5971,7 +5971,7 @@
         "allowMultiple": false
        },
        {
-        "type": "",
+        "type": "string",
```

Comment:
There's too much automagic in queryToObject for me to easily determine whether we split by comma or just expect multiple args of the same name or what.

cc @ncdc